example/mixin/alerts.yaml

```diff
@@ -72,3 +72,15 @@ groups:
     for: 5m
     labels:
       severity: warning
+- name: config-reloaders
+  rules:
+  - alert: ConfigReloaderSidecarErrors
+    annotations:
+      description: Errors encountered while the {{$labels.pod}} config-reloader sidecar
+        attempts to sync config in {{$labels.namespace}} namespace.
```

Comment:
Since we don't have a runbook for this alert, could we add information about potential issues caused by this it to description? Sth like `As a result configuration for service running in {{$labels.pod}} can be stale and cannot be updated anymore.`